### PR TITLE
Fix encoding in feeds [stage-1]

### PR DESCRIPTION
**Pull request by donnapep**

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review. Thx.

**Comment by stulees** [comment]
@donnapep :+1: , looks like a rebase is required though

File: dist/js/settings.js [hunk 1]
```diff
@@ -10968,6 +10969,86 @@ angular.module("risevision.widget.common")
   }]);
 
 angular.module("risevision.widget.common")
+  .factory("googleFontLoader", ["$http", "angularLoad", function ($http, angularLoad) {
```
Comment (rodrigopavezi) [inline dist/js/settings.js]: @donnapep curious, why this has been changed if nothing has changed on the bower js? cheers

Comment (donnapep) [inline dist/js/settings.js]: @rodrigopavezi It was from an earlier pull request that I expected to go first. I've rebuilt. Thx.

**Pull request merged**